cmake/Curl.cmake

```diff
@@ -35,6 +35,8 @@ IF(CMAKE_HOST_WIN32)
   IF(MSVC)
     INSTALL(FILES "buildwin/libcurl.dll" DESTINATION ".")
     INSTALL(FILES "buildwin/zlib1.dll" DESTINATION ".")
+	INSTALL(FILES "buildwin/portaudio_x86.dll" DESTINATION ".")
```

Comment:
While these files indeed should be installed, the Curl.cmake file is certainly the wrong place -- this is not related to culr in any way. Perhaps best handled in CMakeLists.txt.

Reply:
That's my Problem ! I can not realy work with cmake ... I hate it. So ... I don't know how. :